pkg/quota/controller/replenishment_controller.go

```diff
@@ -1,61 +0,0 @@
-package controller
```

Comment:
Should 'we probably want to create a controller' be a post-rebase task in https://github.com/openshift/origin/issues/17428, right?